ui/src/StatusDesktop/Monitoring/Monitor.cpp

```diff
@@ -47,9 +49,36 @@ bool Monitor::isModel(const QVariant &obj) const
     return qobject_cast<QAbstractItemModel*>(obj.value<QObject*>()) != nullptr;
 }
 
-QObject* Monitor::findChild(QObject* obj, const QString& name) const
+QObject* Monitor::findChild(QObject* parent, const QString& name) const
 {
-    return obj == nullptr ? nullptr : obj->findChild<QObject*>(name);
+    if (!parent)
+        return nullptr;
+
+    QSet<QObject*> children;
+
+    for (auto c : parent->children())
+        children << c;
```

Comment:
You can do directly `QSet::fromList(parent->children())` or the QSet range CTOR

Reply:
True, I will fix!